proto/src/google/ground/v1beta1/survey.proto

```diff
@@ -37,6 +37,18 @@ message Survey {
 
   // Required. User id of the survey owner.
   string owner_id = 5;
+
+  // Properties related to data sharing consent.
+  message DataSharingConsent {
```

Comment:
I think "data sharing terms" would be more correct than "consent" here. Consent can be reserved for once the data collectors actually agree to the terms.